server/src/main/java/org/eclipse/openvsx/RegistryAPI.java

```diff
@@ -1716,4 +1716,13 @@ public ResponseEntity<String> getPublicKey(
 
         return ResponseEntity.notFound().build();
     }
-}
\ No newline at end of file
+
+    @GetMapping(path = "/api/version", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Please add `@CrossOrigin` and further API documentation. You can use other `RegistryAPI` endpoints as example.

Reply:
I added everything to the best of my knowledge. I also made a small change to actually report a 404 in case we do not find a registry version (+ respective handling in the webui). I think that is cleaner than just reporting some kind of empty string or something like that.